_includes/pbc/all/install-features/202304.0/install-the-spryker-core-feature.md

```diff
@@ -0,0 +1,962 @@
+
+{% endinfo_block %}
+
+* In case of a single-instance Redis setup, extend your project with the following configuration:
+
+**config/Share/config_default.php**
+
+```php
+<?php
+
+use Spryker\Shared\SessionRedis\SessionRedisConstants;
+
+$config[SessionRedisConstants::ZED_SESSION_REDIS_PROTOCOL] = 'tcp';
+$config[SessionRedisConstants::ZED_SESSION_REDIS_HOST] = '127.0.0.1';
+$config[SessionRedisConstants::ZED_SESSION_REDIS_PORT] = 6379;
+$config[SessionRedisConstants::ZED_SESSION_REDIS_PASSWORD] = false;
+$config[SessionRedisConstants::ZED_SESSION_REDIS_DATABASE] = 2;
+```
+
+If you're using the file system as session storage, extend your project with the following configuration:
+
+**config/Shared/config_default.php**
+
+```php
+<?php
+
+use Spryker\Shared\Session\SessionConfig;
+use Spryker\Shared\Session\SessionConstants;
+use Spryker\Shared\SessionFile\SessionFileConfig;
+use Spryker\Shared\SessionFile\SessionFileConstants;
+
+// ---------- Session
+$config[SessionConstants::ZED_SESSION_SAVE_HANDLER] = SessionFileConfig::SESSION_HANDLER_FILE;
+$config[SessionFileConstants::ZED_SESSION_TIME_TO_LIVE] = SessionConfig::SESSION_LIFETIME_1_HOUR;
+$config[SessionFileConstants::ZED_SESSION_FILE_PATH] = session_save_path();
+```
+
+{% info_block warningBox "Note" %}
+
+All the values in the examples above should be replaced with the real ones used in the corresponding environment.
+
+{% endinfo_block %}
+
+#### Configuration for SecurityBlocker
+
+SecurityBlocker stores the blocked accounts information in Redis. Thus, it needs the connection information to be provided. You can do it in the environment configuration of your project:
+
+**config/Shared/config_default.php**
+
+```php
+$config[SecurityBlockerConstants::SECURITY_BLOCKER_REDIS_PERSISTENT_CONNECTION] = true;
+$config[SecurityBlockerConstants::SECURITY_BLOCKER_REDIS_PROTOCOL] = 'tcp';
+$config[SecurityBlockerConstants::SECURITY_BLOCKER_REDIS_HOST] = '127.0.0.1';
+$config[SecurityBlockerConstants::SECURITY_BLOCKER_REDIS_PORT] = 6379;
+$config[SecurityBlockerConstants::SECURITY_BLOCKER_REDIS_PASSWORD] = false;
+$config[SecurityBlockerConstants::SECURITY_BLOCKER_REDIS_DATABASE] = 7;
+```
+
+Configure the blocking settings for the entity types you want to be blocking. You can set separate settings for a customer (default) and agent. `SecurityBlockerConstants::SECURITY_BLOCKER_BLOCKING_TTL`, `SecurityBlockerConstants::SECURITY_BLOCKER_BLOCK_FOR`, `SecurityBlockerConstants::SECURITY_BLOCKER_BLOCKING_NUMBER_OF_ATTEMPTS` are used as default, so if you use other entity types (like an agent) and do not provide this setting, the defaults will get applied.
+
+`SecurityBlockerConstants::SECURITY_BLOCKER_BLOCKING_TTL` controls the period of time during which the `SecurityBlockerConstants::SECURITY_BLOCKER_BLOCKING_NUMBER_OF_ATTEMPTS` will be counted. In case the number is exceeded, the account will be locked for `SecurityBlockerConstants::SECURITY_BLOCKER_BLOCK_FOR` seconds.
```

Comment:
```suggestion
`SecurityBlockerConstants::SECURITY_BLOCKER_BLOCKING_TTL` controls the period of time during which the `SecurityBlockerConstants::SECURITY_BLOCKER_BLOCKING_NUMBER_OF_ATTEMPTS` will be counted. In case the number is exceeded, the account gets locked for `SecurityBlockerConstants::SECURITY_BLOCKER_BLOCK_FOR` seconds.
```